traitsui/testing/tester/wx/default_registry.py

```diff
@@ -21,5 +28,31 @@ def get_default_registry():
         The default registry containing implementations for TraitsUI editors
         that is wx specific.  
     """
+    registry = get_wobject_registry()
+
+    button_editor.register(registry)
+    
+    return registry
+
+def get_wobject_registry():
```

Comment:
Same question as above, did you want to move this function to helpers?

BTW, unlike [`QObject`](https://doc.qt.io/qt-5/qobject.html),  `wobject` is not a well-known name. Perhaps we should name this `wx_object`?